integration-tests/test-app/index.html

```diff
@@ -3,9 +3,9 @@
 	<head>
 		<meta charset="utf-8" />
 		<style>
-			body { background: #0a0f21; color: #cbcbcb; }
-			input { background-color: inherit; color: inherit; }
-			button { background: inherit; color: inherit; }
+			:root {
+				color-scheme: dark light;
+			}
```

Comment:
Updated the test-app to use browser native dark / light theme. This should respect the system preferences now!

This is also a change we'd like to make to the generated tram-one apps: https://github.com/Tram-One/tram-one-express/issues/110